model/sifrovatko.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+use \Defuse\Crypto\Key,
+  \Defuse\Crypto\Crypto,
+  \Defuse\Crypto\Exception\CryptoException;
+
+class Sifrovatko
+{
+
+
+  /**
+   * @param $text
+   * @return string
+   * @throws \Defuse\Crypto\Exception\EnvironmentIsBrokenException
+   */
+  public function zasifruj($text): string
+  {
+    $key = $this->nactiKlicZKofiguracnihoSouboru();
+    return Crypto::encrypt($text, $key);
+  }
+
+  /**
+   * @param $zasifrovanyText
+   * @return null|string
+   */
+  public function desifruj($zasifrovanyText): string
+  {
+    $key = $this->nactiKlicZKofiguracnihoSouboru();
+    try {
+      return Crypto::decrypt($zasifrovanyText, $key);
+    } catch (CryptoException $e) {
+
+      // An attack! Either the wrong key was loaded, or the ciphertext has
+      // changed since it was created -- either corrupted in the database or
+      // intentionally modified by somebody trying to carry out an attack.
+      return "Text se nepodařilo dešifrovat";
+
+    }
+  }
+
+  /**
+   * @return Key
+   * @throws \Defuse\Crypto\Exception\BadFormatException
+   * @throws \Defuse\Crypto\Exception\EnvironmentIsBrokenException
+   */
+  protected function nactiKlicZKofiguracnihoSouboru()
+  {
+
+    $keyAscii = file_get_contents(__DIR__ . '/../nastaveni/secret-crypto-key.txt');
```

Comment:
Jak se dívám tak ten klíč má nějakých ~130 znaků. To je přijatelná délka, hoď ho prosím do konstanty v nastaveni-local-default.php. V těchto souborech už je vyřešené vše kolem testovacích / ostrých klíčů, přístupů a podobně.